tests/gen_cert_test.py

```diff
@@ -181,3 +179,69 @@ def test_cert_date(self, force_date, locale, timezone, expected):
         """
         cert_date = get_cert_date(force_date, locale, timezone)
         assert_equal(expected, cert_date)
+
+
+class CertificateSubtemplateTests(unittest.TestCase):
+    """
+    Tests the subtemplates feature for certs
+
+    Subtemplates are currently written to key on designations only
+
+    These tests have a specific order
```

Comment:
You'll need to review the three tests.  The comment makes sense given the context.  Maybe there is a better way to accomplish the goal but we'll never get there if the code isn't reviewed.  

The tests do not have any prerequisites.  Nor are they order-dependent.

I disagree with moving the test suite into it's own file.  It's quite small considering there are only three tests,